index.html

```diff
@@ -232,7 +233,7 @@ <h3 class="connect-text">LET'S CONNECT</h3>
           /></a>
         </li>
       </ul>
-      <button class="about-button">Get my resume</button>
+      <button class="about-button">Get my resume,hire me</button>
```

Comment:
Your project is well structured and organized, just a little thing left, make sure the text in your button is the same as that of the template, thanks